src/managers/run-strategies/conditional-creation-strategy.js

```diff
@@ -18,10 +19,10 @@ var Strategy = classFrom(Base, {
     },
 
     /**
-     * Resets current run
-     * @param  {RunService} runService  a Run Service instance for the 'current run' as determined by the Run Manager
-     * @param  {Object} userSession Information about the current user seesion. See AuthManager#getCurrentUserSession for format
-     * @param  {Object} options (Optional) See RunService#create for supported options
+     * Gets a new 'correct' run, or updates the existing one (the definition of 'correct' depends on strategy implementation).
+     * @param  {RunService} runService A Run Service instance for the 'correct' run, as determined by the Run Manager.
```

Comment:
I think "current" instead of "correct" in this case; i.e., strategy determines correctness, run manager determines currentness